package.json

```diff
@@ -71,11 +72,11 @@
   },
   "devDependencies": {
     "@netlify/plugin-nextjs": "^3.9.2",
+    "husky": "^8.0.0",
```

Comment:
Why did you change the dependencies order?

Reply:
To be honest I didn't see that was changed. I'll revert that.